### DISCUSSION TITLE
📊 Advanced Insights

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#770] [New in GitBook]
![Advanced insights](https://github.com/user-attachments/assets/c34379ad-a5f7-4d8d-b3eb-7452224d6f8c)

Keep a closer eye on your documentation’s performance with our **Advanced Insights** update! Track data changes and trends over time to better understand how your content is being used and consumed.

🔍 **What can you do with Advanced Insights?**  
- **Monitor recent data**: See how your site is performing with up-to-date metrics.  
- **Track trends**: Get insights into how user engagement evolves over time.  
- **Coming soon**: Stay tuned for dedicated pages that will dive even deeper into the data.

👉 Check out the updated insights today! Have feedback or suggestions? Start a discussion or drop us a note—we’re excited to hear your thoughts!

📖 https://docs.gitbook.com/published-documentation/insights

**Comment by denisemcg** [comment]
I see there's a new section called Broken URLs.
What's this about? How is it different from the broken links feature? I have a long list (54 items) but not sure what to do about it. 
![image](https://github.com/user-attachments/assets/6ccb564a-e1b2-441e-a8ea-3624b596bb9d)

**Reply by addisonschultz** [reply]
Hey @denisemcg ! Great question - we’re adding this to our next changelog at the moment. It’s a new feature we’ve just shipped that allows you to see broken links from visitors trying to access your docs.

For instance, if someone had clicked a link somewhere on the internet but that link doesn’t exist anymore - if the page was moved or changed. It’s a useful tool to see if there are any redirects that were forgot to be set.

The broken links feature in the editor relates to broken links that are currently found inside your docs (internal links that are broken). 

Hope that clarifies things!

**Reply by denisemcg** [reply]
I don't see how useful this is, e.g. it doesn't provide the source URL or date context. Some of the URLs are gobbledygook so I wonder how they were ever found/used in the first place. Apart from the redirect purpose you mention, what would I use those insights for? Maybe this will be covered in the new docs.

**Reply by addisonschultz** [reply]
There is a chance for spam to be there as well - for instance, if someone visited https://gitbook.com/spam12345 it would show up here

Fixing broken redirects is the main purpose this covers - but it’s good feedback wanting to know the source of the broken link, I’ll pass that to the team

**Reply by addisonschultz** [reply]
@denisemcg one thing you could do as well - in the traffic section of the insights tab, is set a filter that looks for 

“Page in Page not found” - and check the referrer section. This should show you a list of sites where the broken links are coming from. Not a perfect solution, but maybe a workaround for you for now!

**Comment by denisemcg** [comment]
Regarding the Search section, can you include insights on search queries that yielded NO results? I want to know what's missing in my docs.

**Reply by addisonschultz** [reply]
Hi @denisemcg - the search queries should show all searches - including queries that are not found in your docs

**Reply by denisemcg** [reply]
It's good it includes all but it doesn't differentiate them! How do I know if search term X is in my docs or not? I'd like a filter or some kind of success tag, or a warning flag. I have 1000s of items in the search queries list and the ones with no results are the ones I'm interested in at the moment. 
PS Thanks for you prompt replies.
